### ISSUE TITLE
Developer Tools extension button doesn't have a name/aria-label

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#179]
The developer tools "cog icon" is not named. Its being read as simply "link" by SR.
![dev_tools_button_error](https://user-images.githubusercontent.com/93537804/179774431-3ced7ae0-af16-4cab-9dc3-351820d8659d.jpg)
